contracts/RiskManagerV1.sol

```diff
@@ -162,6 +162,11 @@ contract RiskManagerV1 is Auctioneer, Ownable {
             "Address is not a deposit contract"
         );
 
+        require(
+            depositToAuction[depositAddress] == address(0),
+            "Already notified about the deposit"
```

Comment:
Not biggy and up to you:
- how about: "Already notified on the deposit in liquidation"
- if the flow goes through bying with surplus, then technically we still can go through this check > 1 but it will be stopped in `currentState()` check. So my proposition is to put this check below the `curretnState()` check.

Reply:
Good points, updated!